### PR TITLE
Hook LightGBM

### PR DESCRIPTION
**Pull request by rwv**
Tested on Windows 10 & macOS 10.14 with Python 3.7.5
fix https://github.com/pyinstaller/pyinstaller/issues/4155 & https://github.com/pyinstaller/pyinstaller/issues/3863

### CODE REVIEW

**Comment by Legorooj** [comment]
Can you please add a change log entry: https://pyinstaller.readthedocs.io/en/stable/development/changelog-entries.html

**Comment by htgoebel** [comment]
Thank you for this pull-request. For this pull-request to be merged, please 
* reword the **commit messages** to comply to our [Commit Message Rules](https://pyinstaller.readthedocs.io/en/latest/development/commit-messages.html#commit-messages), and
* submit a [**changelog entry**](https://pyinstaller.readthedocs.io/en/latest/development/changelog-entries.html) so our users can learn about your change.
* add the [Copyright header](https://github.com/pyinstaller/pyinstaller/blob/v3.6/COPYING.txt#L72]

File: PyInstaller/hooks/hook-lightgbm.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+# -----------------------------------------------------------------------------
+#
+# A fast, distributed, high performance gradient boosting (GBT, GBDT, GBRT, GBM or MART)
+# framework based on decision tree algorithms, used for ranking, classification and
```
Comment (Legorooj) [inline PyInstaller/hooks/hook-lightgbm.py:13]: Can you shorten these lines to <=79 chars? It'll make travis happy.

**Comment by bjones1** [comment]
I'm removing the ready to merge label until CI passes.

**Pull request merged**